src/components/Table/Table.tsx

```diff
@@ -54,14 +53,14 @@ function getTaskIcon(completed: boolean) {
 function getDifficultyClass(difficulty: string) {
   if (difficulty === "Easy") {
     return styles.easy;
-  } else if (difficulty === "Med.") {
+  } else if (difficulty === "Med") {
     return styles.medium;
   } else {
     return styles.hard;
   }
 }
 
-function Table({ problems }: TableProps) {
+function Table({ problems, solvedProblems }: TableProps) {
```

Comment:
이 컴포넌트가 이 파일에서 가장 중요한 컴포넌트이니 제일 위로 옮겨주시면 중요한 컴포넌트의 코드를 먼저 읽을 수 있어서 도움이 될 것 같습니다. 그리고 밑에 따로 export하기 보다는 함수 선언 부에 export를 붙여주시면 더 빨리 어떤 컴포넌트가 export되는지 알 수 있어서 편할 것 같습니다.

```suggestion
export function Table({ problems, solvedProblems }: TableProps) {
```

Reply:
넵 적용했습니다! 이 부분 고민했던 부분인데 명쾌하게 피드백 주셔서 감사합니다!